From 9344a5b2a8ed7b539d62cce595431b34e28ffcf5 Mon Sep 17 00:00:00 2001
From: Cody Peter Mello <cody.mello@joyent.com>
Date: Thu, 13 Oct 2016 00:56:13 +0000
Subject: [PATCH] DOCKER-949 Adjust set of characters escaped for NAPI Reviewed
 by: Josh Wilsdon <jwilsdon@joyent.com> Approved by: Trent Mick
 <trentm@gmail.com>

---
 lib/backends/sdc/utils.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/backends/sdc/utils.js b/lib/backends/sdc/utils.js
index 4feaa12..7f34502 100644
--- a/lib/backends/sdc/utils.js
+++ b/lib/backends/sdc/utils.js
@@ -982,7 +982,7 @@ function compressPorts(ports) {
 // XXX - workaround for NAPI-367
 function ldapEscape(str) {
     /* JSSTYLED */
-    return str.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, '\\$&');
+    return str.replace(/[=\(\)\*\\]/g, '\\$&');
 }
 
 // ---- exports
-- 
2.21.0

